doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -296,7 +300,7 @@ Here is an `example of a Data Usage Agreement <https://dataverse.org/best-practi
 Restricted Files + Terms of Access 
 ----------------------------------
 
-If you restrict any files in your dataset, you will be prompted by a pop-up to enter Terms of Access for the data. This can also be edited in the Terms tab or selecting Terms in the "Edit" dropdown button in the dataset. You may also allow users to request access for your restricted files by enabling "Request Access". To add more information about the Terms of Access, we have provided fields like Data Access Place, Availability Status, Contact for Access, etc.
+If you restrict any files in your dataset, you will be prompted by a pop-up to enter Terms of Access for the data. This can also be edited in the Terms tab or selecting Terms in the "Edit" dropdown button in the dataset. You may also allow users to request access for your restricted files by enabling "Request Access". To add more information about the Terms of Access, we have provided fields like Data Access Place, Availability Status, Contact for Access, etc. If you restrict a file, it will not have a preview shown on the file page. 
```

Comment:
See the comment above for how "If you restrict a file, it will not have a preview shown on the file page" is not entirely accurate.

Reply:
reworded in https://github.com/IQSS/dataverse/commit/a992e0efe0ab817bc0afec5b9bf355868440e72a

Reply:
Talked about in standup, this is all set.